tagging_intents_by_intent_name

```diff
@@ -0,0 +1,117 @@
+"""
+tagging intents by using intent name as the search string
+
+Set HF_USERNAME and HF_PASSWORD as environment variables
+"""
+# ******************************************************************************************************************120
+
+# third part imports
+import click
+import humanfirst
+
+
+@click.command()
+@click.option('-d', '--delimiter', type=str, default="-",
+              help='Delimiter used in intent name')
+@click.option('-a', '--add_tag', type=str, default="",
+              help='Add tag names')
+@click.option('-d', '--delete_tag', type=str, default="",
+              help='Delete tag names')
+@click.option('-s', '--search_string', type=str, required=True,
+              help='search string in intent_name')
+@click.option('-u', '--username', type=str, default='',
+              help='HumanFirst username if not setting HF_USERNAME environment variable')
+@click.option('-p', '--password', type=str, default='',
+              help='HumanFirst password if not setting HF_PASSWORD environment variable')
+@click.option('-n', '--namespace', type=str, required=True,
+              help='HumanFirst namespace')
+@click.option('-b', '--playbook', type=str, required=True,
+              help='HumanFirst playbook id')
+def main(delimiter: str, add_tag: str,
+         delete_tag: str, search_string: str,
+         username: str, password: str,
+         namespace: str, playbook: str):
+    '''Main function'''
+
+    if add_tag == "" and delete_tag == "":
+        error_msg = "Both add_tag and delete_tag are empty. Atleast one should be provided"
+        raise  Exception(error_msg) # pylint: disable=broad-exception-raised
```

Comment:
Something I found out recently.  If you raise your custom exceptions as RunTimeErrors then you don't need to declare them explicitly and will pass this check
https://docs.python.org/3/library/exceptions.html

```
exception RuntimeError
Raised when an error is detected that doesn’t fall in any of the other categories. The associated value is a string indicating what precisely went wrong
```